contracts/governance/AxelarServiceGovernance.sol

```diff
@@ -81,7 +81,7 @@ contract AxelarServiceGovernance is InterchainGovernance, MultisigBase, IAxelarS
         }
```

Comment:
With the check above the last else-if can just become an else because we know at that point the commandID must be 3 or it would have reverted. Right now with the else-if test coverage cannot reach 100% because it wants to check the case where the last condition fails, but that is not possible. Unless we want to keep it like this for readability.

Reply:
how about instead of casting `commandId` to an enum, we convert the enums to uint256 during the if comparison, and thus add an `else` clause that reverts. We won't need the explicit check for greater max enum value anymore and it's easier for coverage and slither to reason about.

Reply:
I implemented the change, let me know if this is what you were referring to